core/src/repair/quic_endpoint.rs

```diff
@@ -408,11 +408,16 @@ async fn handle_connection(
     ));
     match futures::future::try_join(send_requests_task, recv_requests_task).await {
         Err(err) => error!("handle_connection: {remote_pubkey}, {remote_address}, {err:?}"),
-        Ok(((), Err(err))) => {
-            debug!("recv_requests_task: {remote_pubkey}, {remote_address}, {err:?}");
-            record_error(&err, &stats);
+        Ok(out) => {
+            if let (Err(ref err), _) = out {
+                error!("send_requests_task: {remote_pubkey}, {remote_address}, {err:?}");
```

Comment:
I think this condition might be expected due to network issues or validator restart? Maybe debug is enough like before?

Reply:
changed to `debug!`.